WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -379,12 +379,19 @@ + (NSArray *)routes
     return FBResponseWithErrorFormat(@"The element is expected to be a valid Picker Wheel control. '%@' was given instead", element.wdType);
   }
   NSString* order = [request.arguments[@"order"] lowercaseString];
+  CGFloat offset = (CGFloat)0.2;
```

Comment:
last time the compiler didn't like that w/o explicit typecast, since CGFloat is a typedef for float on x86 and for double on x64